src/mds/Server.cc

```diff
@@ -1122,8 +1128,11 @@ void Server::recover_filelocks(CInode *in, bufferlist locks, int64_t client)
  * to trim some caps, and consequently unpin some inodes in the MDCache so
  * that it can trim too.
  */
-void Server::recall_client_state(void)
+void Server::recall_client_state(double ratio, bool report_acks)
 {
+  MClientSession *m;
```

Comment:
Why did you move this var?

Reply:
Moved it during re-writings. Undid this change.